faiss/impl/ScalarQuantizer.cpp

```diff
@@ -40,7 +41,7 @@ namespace faiss {
  ********************************************************************/
 
 #ifdef __AVX2__
-#ifdef __F16C__
+#if defined(__F16C__) || defined(_MSC_VER)
```

Comment:
Realizing now that `__F16__` could be set in `platform_macros.h` like `__SSEx__`...